session/common.go

```diff
@@ -308,6 +310,16 @@ func (x *commonData) SetAuthKey(key neofscrypto.PublicKey) {
 	x.authKey = x.authKey[:key.Encode(x.authKey)]
 }
 
+// SetIssuer allows to set issuer before Sign call.
+// Using this method is not required when Sign is used (issuer will be derived from the signer automatically).
+// When using it please ensure that the token is signed with the same signer as the issuer passed here.
+func (x *commonData) SetIssuer(id user.ID) error {
```

Comment:
Given it's this simple now and given that other setters return nothing, I'd suggest doing the same here.

Reply:
Oh, yes, this part left from previous iteration. Removed